build.gradle

```diff
@@ -719,22 +719,30 @@ defineProperty("TEST_ONLY", "false")
 ext.IS_TEST_ONLY = Boolean.parseBoolean(TEST_ONLY) || !buildSdkForTest
 
 ext.IS_TEST_SDK = false
+ext.TEST_SDK_DIR = ""
+
 if (hasProperty("TEST_SDK_PATH")) {
     if (!IS_TEST_ONLY) {
         fail("TEST_ONLY must be set to true when using an external TEST_SDK_PATH for running tests: include -PTEST_ONLY=true")
     }
-    def sdkShimsPath = file("${TEST_SDK_PATH}")
-    def sdkPath = file("${TEST_SDK_PATH}/sdk")
-    def shimsPath = file("${TEST_SDK_PATH}/shims")
+    String testSdkPath = "${TEST_SDK_PATH}"
+    if (testSdkPath.startsWith(".")) {
+        testSdkPath = file(testSdkPath).absolutePath
+    }
+    ext.TEST_SDK_DIR = testSdkPath
+    def sdkShimsPath = file("${TEST_SDK_DIR}")
+    def sdkPath = file("${TEST_SDK_DIR}/sdk")
+    def shimsPath = file("${TEST_SDK_DIR}/shims")
     if (!sdkShimsPath.exists() || !sdkShimsPath.isDirectory() ||
              !sdkPath.exists() || !sdkPath.isDirectory() ||
            !shimsPath.exists() || !shimsPath.isDirectory()) {
         fail("The provided TEST_SDK_PATH=${TEST_SDK_PATH} is invalid")
     }
     ext.IS_TEST_SDK = true
 } else {
-    defineProperty("TEST_SDK_PATH", "${rootProject.buildDir}")
+    ext.TEST_SDK_DIR = "${rootProject.buildDir}"
 }
+println "TEST_SDK_PATH: " + TEST_SDK_DIR
```

Comment:
Minor: Consider moving this down to the block where most other build properties like this are logged?

Reply:
Moved to use `logger.quiet` along with other properties.